docs/sources/operations/storage/tsdb.md

```diff
@@ -0,0 +1,77 @@
+---
+title: TSDB
+---
+# TSDB Index Store
+
+:warning: TSDB is still an experimental feature. It is not recommended to be used in production environments.
+
+With TSDB index store, you can run Grafana Loki without any dependency on NoSQL stores for storing index.
+It works the same way as [boltdb-shipper](./boltdb-shipper) index store, but with reduced TCO and better query performance.
+Loki uses [Prometheus TSDB format](https://github.com/prometheus/prometheus/tree/main/tsdb#tsdb) for storing the index with 
```

Comment:
I just wanted to put it out there since it establishes the premise that we are using a known and tested index format and indirectly gives the due credit to Prometheus.